### PR TITLE
setting up landing page demo and refactoring molecules/organisms

### PR DESCRIPTION
**Pull request by kave**
Refactored the way we create prototypes, so they are contained within the molecules and helps readability of atomic element with less duplication of code in the if cases

@jimmynotjim 
@KimberlyMunoz 
@anselmbradford 
@sebworks

### CODE REVIEW

**Comment by sebworks** [comment]
Why can't just populate the database (via a json dump or migrations) which would make this unnecessary?

**Comment by kave** [comment]
@sebworks thats what we originally had, but keeping that dump file upto date with our constant DB changes was becoming a nightmare. we can circle back after we have gone live to see if it makes sense to pursue it

File: cfgov/jinja2/v1/learn-page/index.html [hunk 1]
```diff
@@ -12,6 +11,9 @@
 
 {% set breadcrumb_items = [('#', '#', 'Sample Breadcrumb')] %}
 
+{% if not page %}
+  {% set prototype = true %}
```
Comment (jimmynotjim) [inline cfgov/jinja2/v1/learn-page/index.html]: Indent 4 spaces

**Comment by jimmynotjim** [comment]
I can dig it :+1:

**Comment by sebworks** [comment]
:+1:

**Pull request merged**